synopsys-task/task.json

```diff
@@ -102,7 +102,7 @@
       "type": "string",
       "label": "Polaris Application",
       "defaultValue": "",
-      "required": true,
```

Comment:
Can we think of adding something in helpMarkDown like: Application name in Polaris. If not provided, will use azure repository name as default value.
Same in case of other scenarios for classic editor. Please check with team as well.

Reply:
Updated the helpMarkDowns for necessary fields of classic editor. Please check them.